projects/plugins/jetpack/changelog/fix-php82-dynamic-properties-masterbar

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Offtopic: This sort of PR is why semver probably doesn't make sense for Jetpack-the-plugin. Is this random field on this class part of a public API? If so, making it private should be 'major' significance. But chances are it's not supposed to be.

Anyway, I see nothing outside this class using it in the monorepo, or in wpdirectory, or in a8c internal codebases.